File: problems/problems_2528/solution.go
package problem2528

import (
	"encoding/json"
	"log"
	"strings"
)

func maxPower(stations []int, r int, k int) int64 {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var stations []int
	var r int
	var k int

	if err := json.Unmarshal([]byte(inputValues[0]), &stations); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &r); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[2]), &k); err != nil {
		log.Fatal(err)
	}

	return maxPower(stations, r, k)
}

File: problems/problems_2528/solution.ts
function maxPower(stations: number[], r: number, k: number): number {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const stations: number[] = JSON.parse(inputValues[0]);
	const r: number = JSON.parse(inputValues[1]);
	const k: number = JSON.parse(inputValues[2]);
	return maxPower(stations, r, k);
}
